README.md

```diff
@@ -112,15 +133,15 @@ This project has been tested against the following versions:
 ## Caveats
 
 Cassandra is an eventually consistent, AP database, and so applying schema updates are not as simple
-as a strongly consistent database.
+as a traditional relational database.
 
 * Certain schema changes can cause data corruption on cassandra. Be very careful changing a schema for a
   table that is being actively written to. Generally, adding columns is safe (but be careful with
-  collection types). Test your migration before rolling out to production.
+  collection types). Test the migration before rolling out to production.
 
-* AP properties of Cassandra also apply to schema updates - so it is possible for your cluster to have an
-  inconsistent schema across nodes in case of split brain or other situation. We use `ConsistencyLevel.ALL`
-  to try and alleviate this.
+* AP properties of Cassandra also apply to schema updates - so it is possible for a cluster to have an
+  inconsistent schema across nodes in case of split brain or other situation. `cqlmigrate` tries to
+  alleviate this with appropriate consistency levels.
```

Comment:
Do we want to be specific about what consistency levels are used, or just let people read through the code to find out?